viam-orb-slam3_test.go

```diff
@@ -483,6 +477,8 @@ func createSLAMService(
 	attrCfg *slamConfig.AttrConfig,
 	model string,
 	logger golog.Logger,
+	// TODO(zack) will be fixed with integration tests in the next few PRs
```

Comment:
[nit] TODO should use a jira ticket number instead of a person's name.

Reply:
Got it. That makes sense.